src/complex/Parameters/MultiPathSelectionParameter.cpp

```diff
@@ -0,0 +1,136 @@
+#include "MultiPathSelectionParameter.hpp"
+
+#include <fmt/core.h>
+
+#include <nlohmann/json.hpp>
+
+#include "complex/Common/Any.hpp"
+
+namespace complex
+{
+MultiPathSelectionParameter::MultiPathSelectionParameter(const std::string& name, const std::string& humanName, const std::string& helpText, const ValueType& defaultValue)
+: MutableDataParameter(name, humanName, helpText, Category::Required)
+, m_DefaultValue(defaultValue)
+{
+}
+
+Uuid MultiPathSelectionParameter::uuid() const
+{
+  return ParameterTraits<MultiPathSelectionParameter>::uuid;
+}
+
+IParameter::AcceptedTypes MultiPathSelectionParameter::acceptedTypes() const
+{
+  return {typeid(ValueType)};
+}
+
+nlohmann::json MultiPathSelectionParameter::toJson(const std::any& value) const
+{
+  const auto& paths = GetAnyRef<ValueType>(value);
+  nlohmann::json json = nlohmann::json::array();
+  for(const auto& path : paths)
+  {
+    json.push_back(path.toString());
```

Comment:
```suggestion
    json.push_back(std::move(path.toString()));
```
I assume the string being returned is being assembled in the function and returned as an rvalue thus std::move ensures its an in-place move-construction preventing loss of efficiency